tests/kafkatest/tests/streams/base_streams_test.py

```diff
@@ -44,8 +44,9 @@ def get_producer(self, topic, num_messages, repeating_keys=None):
                                   self.kafka,
                                   topic,
                                   max_messages=num_messages,
-                                  acks=1,
-                                  repeating_keys=repeating_keys)
+                                  acks=self.num_brokers,
```

Comment:
Another measure to improve resilience: make sure that messages produced during test setup are acknowledged by all brokers so we can be sure that during broker bounce operations we'll never fail a test due to expectedly unreplicated data.